phy/cluster/algorithms.py

```diff
@@ -18,7 +18,9 @@
 from ..utils._types import Bunch
 from ..utils.event import EventEmitter, ProgressReporter
 from ..utils.logging import debug, info
-from ..electrode.mea import _channels_per_group, _probe_adjacency_list
+from ..electrode.mea import (_channels_per_group,
```

Comment:
is there any reason for this, out of interest? or just a leftover from adding and removing some import?